intranet/apps/announcements/notifications.py

```diff
@@ -113,9 +115,13 @@ def announcement_posted_email(request, obj, send_all=False):
     if settings.EMAIL_ANNOUNCEMENTS:
         subject = "Announcement: {}".format(obj.title)
         if send_all:
-            users = get_user_model().objects.all()
+            users = get_user_model().objects.exclude(Q(user_type="service") | Q(graduation_year__gte=get_senior_graduation_year()))
```

Comment:
Do we want to include only students, teachers, and counselors (other user types that might not be supposed to receive these emails, like `user` or `tjstar` users, would still receive them under this)?

Also, this would exclude current seniors. Maybe something like

```suggestion
            users = get_user_model().objects.filter(user_type__in=["student", "teacher", "counselor"], graduation_year__lte=get_senior_graduation_year())
```